### PR TITLE
Project model

### PR DESCRIPTION
**Pull request by pjanik**
Adds `Admin::Project` model. It has name, landing page slug and landing page content at the moment.

Main features:
- activities, external activities and investigations can be assigned to project.
- material search results can be filtered by project.
- project can have its own landing page, available at custom ULR defined by slug (`portal.concord.org/<project-slug>`).

Hopefully it's well tested, including cucumber tests.

### CODE REVIEW

**Comment by knowuh** [comment]
This is looking great.

**Comment by pjanik** [comment]
Thanks Noah!
@psndcsrv, do you want to double check or should I merge?

File: db/schema.rb [hunk 1]
```diff
@@ -845,6 +867,21 @@
...
+    t.integer  "height"
+    t.float    "scale"
+    t.string   "image_url"
+    t.integer  "user_id"
+    t.string   "credits"
+    t.string   "publication_status"
+    t.datetime "created_at",         :null => false
+    t.datetime "updated_at",         :null => false
+  end
+
```
Comment (psndcsrv) [inline db/schema.rb]: I think this create_table statement got included accidentally -- I don't see a migration for it.

**Comment by psndcsrv** [comment]
That extra table definition in db/schema.rb was the only thing I noticed. Everything else looks good!

File: db/schema.rb (hunk 1 above)
Comment (scytacki) [inline db/schema.rb]: that probably came from looking at this pull request locally: #56

**Comment by pjanik** [comment]
Thanks! Fixed, so I'm merging it.

**Pull request merged**